crates/pbh-ctf/src/config.rs

```diff
@@ -0,0 +1,46 @@
+use std::path::Path;
+
+use reqwest::Url;
+use serde::Deserialize;
+
+pub const CONFIG_PREFIX: &str = "CTF_BOT";
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct CtfConfig {
+    /// Ethereum Signer
+    pub private_key: String,
+    /// The Wallet Nonce
+    pub wallet_nonce: u64,
+    /// The PBH nonce limit
+    pub pbh_nonce_limit: u8,
+    /// Semaphore secret
+    pub secret: String,
+    /// WC Sepolia Provider
+    pub provider: Url,
+    /// The start timestamp of the game
+    pub start_timestamp: u64,
```

Comment:
We should add this to the contract as a getter function so that we remove the possibility of user error.